en/vespa8-release-notes.html

```diff
@@ -893,6 +927,90 @@ <h2 id="removed-metrics">Removed metrics</h2>
       <td>Use <em>http.status.4xx.rate</em> with dimension <em>statusCode</em>==403</td>
     </tr>
+    <tr>
+      <td><em>vds.visitor.allthreads.averagevisitorlifetime.sum.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.visitor.allthreads.averagequeuewait.sum.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.visitor.allthreads.queuesize.sum.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.visitor.allthreads.completed.sum.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.visitor.allthreads.averagemessagesendtime.sum.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.visitor.allthreads.averageprocessingtime.sum.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.filestor.alldisks.queuesize.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.filestor.alldisks.averagequeuewait.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.filestor.alldisks.allthreads.put.sum.latency.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.filestor.alldisks.allthreads.remove.sum.latency.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.filestor.alldisks.allthreads.get.sum.latency.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.filestor.alldisks.allthreads.update.sum.latency.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.filestor.alldisks.allthreads.createiterator.sum.latency.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.filestor.alldisks.allthreads.visit.sum.latency.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.filestor.alldisks.allthreads.remove_location.sum.latency.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.filestor.alldisks.allthreads.deletebuckets.sum.latency.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.distributor.puts.sum.latency.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.distributor.removes.sum.latency.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.distributor.updates.sum.latency.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.distributor.gets.sum.latency.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.distributor.visitor.sum.latency.average</em></td>
```

Comment:
These all seem to use the legacy metric names (including `.sum.`). Should we use specify the new ones instead? Or both?

Reply:
I think that using strikethrough should be clear enough.
If anyone has used any of these metrics, they are very likely to infer the action required.